x/did/client/rest/tx.go

```diff
@@ -12,38 +14,37 @@ import (
 	"github.com/ixofoundation/ixo-blockchain/x/did/internal/keeper"
 	"github.com/ixofoundation/ixo-blockchain/x/did/internal/types"
 	"github.com/ixofoundation/ixo-blockchain/x/ixo"
+
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
 )
 
 func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/did", createDidRequestHandler(cliCtx)).Methods("POST")
 	r.HandleFunc("/credential", addCredentialRequestHandler(cliCtx)).Methods("POST")
```

Comment:
Could you please rename any of these URLs that you see to the following format please?

- `moduleName/actionName`

So in this case we will have the following (note that we use snake_case not camelCase here):

- `did/create_did`
- `did/add_credential`

Reply:
modified and noted